### ISSUE TITLE
PHP 7 compatibility issue in passbolt update / migration scripts

### ISSUE THREAD

**Issue by K0n24d** [passbolt/passbolt_api#29]
When running migration for v1.2.0 I get the following error message:

```
Cake Migration Shell
---------------------------------------------------------------
Running migrations:
  [1474629203] 1474629203_Migration_1.2.0
PHP Fatal error:  Cannot use 'String' as class name as it is reserved in /var/www/passbolt/lib/Cake/Utility/String.php on line 26

Fatal error: Cannot use 'String' as class name as it is reserved in /var/www/passbolt/lib/Cake/Utility/String.php on line 26
Fatal Error Error: Cannot use 'String' as class name as it is reserved in [/var/www/passbolt/lib/Cake/Utility/String.php, line 26]

```

Looks like an issue with php 7 for me...

**Comment by stripthis** [comment]
Yes. This seems to me to be an issue with the plugin we use to manage migrations.
https://github.com/CakeDC/migrations

We need to to update the library using composer to see if the latest version fix this, and if not open an issue / make a PR to CakeDC.

**Comment by stripthis** [comment]
ref.     PASSBOLT-1784

**Comment by K0n24d** [comment]
Looking at https://github.com/cakephp/cakephp/issues/7573 it seems this is not CakeDC specific but related to CakePHP, and has been solved in 3.X and backported into 2.7 (see https://github.com/cakephp/cakephp/pull/5559)

Also looking at https://github.com/CakeDC/migrations/pull/258 CakeDC/migrations has renamed String to CakeText (as needed with previous backport) but from what i see this is only in the develop branch for now. So updating to a newer release won't probably work for now.

**Comment by stripthis** [comment]
@K0n24d it has actually been merged, we'll publish a fix for this with the release tomorrow or day after. Thanks!

**Comment by stripthis** [comment]
Fixed with v1.3.1
